Client/src/main/java/de/baleipzig/pdfextraction/client/utils/SceneHandler.java

```diff
@@ -34,9 +33,15 @@
             final Parent parent = FXMLLoader.load(newView.getFXML(), this.languageHandler.getCurrentBundle(), null, Injector::createInstance, StandardCharsets.UTF_8);
             final Scene scene = new Scene(parent);
             current.setScene(scene);
-            current.show();
 
-            currentView = newView;
+            // dont set currentView to substages, this will result in a bug at reloadScene
+            if (current.getOwner() != null) {
+                current.initModality(Modality.WINDOW_MODAL);
+                current.showAndWait();
+            } else {
+                current.show();
+                currentView = newView;
+            }
```

Comment:
- `showAndWait` blockiert den aktuellen Thread wie der Name schon sagt, ist das gewollt?
 - Was soll die unterteilung hier bringen?

Reply:
das bringt, das man nicht mehr an die "primary stage" ran kommt wenn das About fenster offen ist, ich hab mich dabei an der Umsetzung in Intellij orientiert

die Unterteilung ist realtiv wichtig, wenn man currentView auf die About View setzt kommt es zu einem Bug wenn man die Sprache Ändern / die Stage reloaden will

Reply:
- siehe Kommentar über der Unterteilung :D